src/backupNg/backupNg.spec.js

```diff
@@ -384,4 +384,192 @@ describe("backupNg", () => {
     });
     expect(vmTask.data.id).toBe(vmId);
   });
+
+  test("execute three times a delta backup with 2 remotes, 2 as retention, 2 as fullInterval", async () => {
+    jest.setTimeout(6e4);
+    const nfsId = await xo.createTempRemote(config.remotes.nfs);
+    const smbId = await xo.createTempRemote(config.remotes.smb);
+
+    const vmId = await xo.createTempVm({
+      name_label: "XO Test Temp",
+      name_description: "Creating a temporary vm",
+      template: config.templates.default,
+      VDIs: [
+        {
+          size: 1,
+          SR: config.srs.default,
+          type: "user",
+        },
+      ],
+    });
+
+    const scheduleTempId = randomId();
+    const { id: jobId, settings } = await xo.createTempBackupNgJob({
+      ...defaultBackupNg,
+      mode: "delta",
+      schedules: {
+        [scheduleTempId]: DEFAULT_SCHEDULE,
+      },
+      settings: {
+        "": {
+          reportWhen: "never",
+          fullInterval: 2,
+        },
+        [nfsId]: { deleteFirst: true },
+        [scheduleTempId]: { exportRetention: 2 },
+      },
+      vms: {
+        id: vmId,
+      },
+    });
+
+    const schedule = await xo.getSchedule({ jobId });
+    expect(typeof schedule).toBe("object");
+
+    const operationalRemote = [];
+    let testOnNfs = true;
+    await xo.call("remote.test", { id: nfsId }).then(
```

Comment:
If a remote not works, the tester should be notified. To keep it simple, we can let SMB optional. If it not exists we backup only on the NFS.